views/ifsc.erb

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Razorpay IFSC Toolkit</title>
+    <link href='/main.css' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Source+Code+Pro' rel='stylesheet' type='text/css'>
+    <link rel="shortcut icon" href="https://razorpay.com/favicon.png" type="image/png">
+  </head>
+  <body class="home-template">
+    <header class="main-header">
+    <ul id="nav">
+      <div class="container">
+        <li><a id="logo-link" href="https://razorpay.com/?ref=ifsc"></a></li>
+        <li><a href="https://razorpay.com/blog/">Blog</a></li>
+        <div class="middle-links">
+          <li><a href="https://razorpay.com/features/?ref=ifsc">Features</a></li>
+          <li><a href="https://razorpay.com/demo/?ref=ifsc">Demo</a></li>
+          <li><a href="https://docs.razorpay.com" target="_blank">Docs</a></li>
+        </div>
+        <div class="float-right">
+          <li><a href="https://dashboard.razorpay.com/#/access/signin" class="login-link">Log In</a></li>
+          <li><a href="https://dashboard.razorpay.com/#/access/signup?ref=ifsc" class="btn login-link">Sign Up</a></li>
+        </div>
+      </div>
+    </ul>
+    </header>
+    <div class='container'>
+      <table class="ifsc-table">
+        <tr>
+          <th>IFSC</th>
+          <td><%= data['IFSC']%></td>
+        </tr>
+        <tr>
+          <th>Address</th>
+          <td><%= data['ADDRESS']%></td>
```

Comment:
also, add https://schema.org/ context schema so that search engines recognize the data easily